lib/tasks/backfill_in_person_pending_at.rake

```diff
@@ -75,4 +75,67 @@ namespace :profiles do
 
     warn "backfill_in_person_verification_pending_at left #{profiles.count} rows"
   end
+
+  ##
+  # Usage:
```

Comment:
Checked on 09/04/24 at approx. 11:30ET
- Expired (3786), Cancelled *813), Failed (587) - Total 5186 in prod
- Establishing 1313